frontend/src/metabase/query_builder/components/view/ViewHeader/components/FilterHeaderButton.unit.spec.tsx

```diff
@@ -22,21 +22,20 @@ const QUERY_WITH_FILTERS = Question.create({
     database: SAMPLE_DB_ID,
     query: {
       "source-table": ORDERS_ID,
+      filter: [
+        "and",
+        ["time-interval", ["field", ORDERS.CREATED_AT, null], -30, "day"],
```

Comment:
we shouldn't be using refs like this in unit tests, should we?

Reply:
Yes, for sure. This should be migrated as well.

Reply:
I'll defer this to other PRs